app/controllers/users_controller.rb

```diff
@@ -4,5 +4,8 @@ def show
     @user = User.find_by!(username: params.fetch(:username))
   end
 
-end
+  def liked
+    @user = User.find_by!(username: params.fetch(:username))
+  end
 
```

Comment:
Great job here, 
Similar to show and liked, try getting the same tabbed interface working for the remaining routes.